Chris4gl openedge117

chris4gl opened the pull request.

chris4gl commented:
I have uploaded the docsets for OpenEdge:  

OE 11.7: https://www.progresstalk.com/docset/OpenEdge117.docset.tgz
OE 11.6: https://www.progresstalk.com/docset/OpenEdge116.docset.tgz
OE 11.5: https://www.progresstalk.com/docset/OpenEdge115.docset.tgz

Thanks!

Kapeli commented:
One of the docsets failed validation:
```
Checking OpenEdge116/OpenEdge116.docset.tgz...
File not found at "aswsp/compiletime-code.html" for entry with row id "953" of "OpenEdge116.docset"
File not found at "aswsp/compiletime-syntax-elements.html" for entry with row id "954" of "OpenEdge116.docset"
File not found at "aswsp/runtime-code.html" for entry with row id "955" of "OpenEdge116.docset"
File not found at "aswsp/runtime-syntax-elements.html" for entry with row id "956" of "OpenEdge116.docset"
File not found at "aswsp/how-compiletime-and-runtime-code-interact.html" for entry with row id "957" of "OpenEdge116.docset"
File not found at "aswsp/using-the-webspeeddefined-output-stream.html" for entry with row id "995" of "OpenEdge116.docset"
File not found at "aswsp/twophase-commit.html" for entry with row id "1080" of "OpenEdge116.docset"
```

Also, why did you set up separate docsets for these instead of updating the current [OpenEdge docset](https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets/OpenEdge)? You can update that docset to 11.7 and add 11.6 and 11.5 as specific versions.

chris4gl commented:
I went and updated the existing OpenEdge docset as you recommended, and added specific versions for each docset.  I also fixed the validation errors in the 11.6 docset, and made some more tweaks to the 11.7 docset as well.

These are available here:
https://www.progresstalk.com/docset/OpenEdge.11.5.1.docset.tgz
https://www.progresstalk.com/docset/OpenEdge.11.6.4.docset.tgz
https://www.progresstalk.com/docset/OpenEdge.11.7.2.docset.tgz

Thanks,
Chris
